tests/modules/adapterremoval/test.yml

```diff
@@ -1,31 +1,44 @@
 - name: adapterremoval test_adapterremoval_single_end
-  command: nextflow run ./tests/modules/adapterremoval -entry test_adapterremoval_single_end -c ./tests/config/nextflow.config -c ./tests/modules/adapterremoval/nextflow.config
+  command: nextflow run tests/modules/adapterremoval -entry test_adapterremoval_single_end -c tests/config/nextflow.config
   tags:
     - adapterremoval
   files:
-    - path: output/adapterremoval/test.log
+    - path: /tmp/tmppy465nmo/adapterremoval/test.discarded.gz
```

Comment:
I guess this may be the reasons the tests fail. Had to replace the `/tmp..` with `output` manually as well in some modules

Reply:
Ah crap... I forgot each test gets its own tmp directory 🤦🏼‍♀️.

Yeah probably why tests failing, will fix later!